src/main/java/Duke.java

```diff
@@ -8,21 +8,39 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
 
-        String line = "_________________________________\n";
+        String line = "____________________________________________________\n";
```

Comment:
Naming can be changed to avoid confusion down the code block, could change to  -> e.g. 'static final String margin'  (since its a repeating string pattern to be printed out)